tests/test_cases/truncate.c

```diff
@@ -0,0 +1,72 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <fcntl.h>
+#include <sys/stat.h>
+#include <unistd.h>
+
+const char *FILENAME = "testfile.txt";
```

Comment:
I think since were having problems with the auxiliary files its worth having the testfile be like truncatefile.txt

Reply:
Will also have to add the testfile to the suite.

Reply:
Done